hoot-core/src/main/cpp/hoot/core/io/HootNetworkRequest.cpp

```diff
@@ -89,9 +89,6 @@ bool HootNetworkRequest::_networkRequest(QUrl url, const QMap<QNetworkRequest::K
                                          QNetworkAccessManager::Operation http_op,
                                          const QByteArray& data)
 {
-  //  Disable logging for the QNetworkAccessManager calls because it logs an error when
-  //  run in a sub-thread.  An exception is thrown below for error handling instead of logging
-  boost::shared_ptr<DisableLog> disable(new DisableLog());
```

Comment:
Turning off logging was blocking the progress and some error logging in the OSM API writing.  It isn't necessary.